src/riscv_opcodes/go_utils.py

```diff
@@ -9,12 +8,19 @@
 logging.basicConfig(level=logging.INFO, format="%(levelname)s:: %(message)s")
 
 
-def make_go(instr_dict: InstrDict):
-
-    args = " ".join(sys.argv)
-    prelude = f"""// Code generated by {args}; DO NOT EDIT."""
-
-    prelude += """
+def make_go(instr_dict: InstrDict, extensions: list[str]):
+    comment = """// Code generated with following arguments; DO NOT EDIT.
+/*
+make inst.go"""
+    if extensions:
+        comment += " EXTENSIONS='\\\n"
+        n = 4
+        exts = sorted(extensions)
```

Comment:
I am not sure with the sorting of extension here as if suppose a tool relies on a particular order _(for readability, or minimal diff when adding one extension)_, this change may cause larger diffs than necessary each time a new extension is added.